spam/02_spam_data_augmentation_tutorial.py

```diff
@@ -217,22 +231,25 @@ def replace_adjective_with_synonym(x):
             break
 pd.DataFrame(transformed_examples)
 
+# %% [markdown]
+# We notice a couple of things about the TFs.
+# * Sometimes they make trivial changes (_titles_ -> _title_ for replace_noun_with_synonym). This can still be helpful for training our model, because it teaches the model that singular and plural versions of words have similar meanings.
+# * Sometimes they make the sentence less meaningful (e.g. swapping _young_ and _more_ for swap_adjectives).
+#
+# Data augmentation can be tricky for text inputs, so we expect most TFs to be a little flawed. But TFs can still be useful despite the flaws if they add enough good augmentations.
```

Comment:
maybe link to papers (e.g. https://arxiv.org/abs/1901.11196) that say that augmentations don't need to be high quality/correct to be useful for modeling